README.md

```diff
@@ -1,3 +1,15 @@
+# What is this?
+This is my first time using Github so don't get too upset if I've done something wrong :)
+
+I've only created this branch to share my docker-compse.yml and .env files in order to help the people who want to use this awesome image using docker compose.
```

Comment:
The readme is shown in the first page. I'd remove this part as it would not make sense for the users.